release/assets/macos_cli_installation.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
+USER_PROVIDED_PATH=$1
+
+main() {
+  cat << EOF
+   ______
+  / ____/___  ____    ( )_  _______
+ / /   / __ \/ __ \  / / / / / ___/
+/ /___/ /_/ / / / / / / /_/ / /
+\____/\____/_/ /_/_/ /\____/_/
+                /___/
+EOF
+
+  if [[ $USER_PROVIDED_PATH = "--help" ]]; then
+    print_help
+  fi
+  executable_path=${USER_PROVIDED_PATH:-'/usr/local/bin'}
+
+  echo "Configuring the Conjur CLI. This will take a few moments...."
+  create_symbolic_link $executable_path
+
+  conjur --help > /dev/null
+  echo "Successfully configured the Conjur CLI. To get started, run 'conjur --help'"
+}
+
+create_symbolic_link() {
+  # Creates symbolic link to the path of the user's choosing. Default location: '/usr/local/bin'
```

Comment:
can we move this to live above the method name?

Reply:
"Creates symbolic link to the path of the user's choice."